Live/Classes/GameObject.cpp

```diff
@@ -0,0 +1,29 @@
+#include "GameObject.h"
+
+USING_NS_CC;
+
+GameObject::GameObject(const std::string& file_name, unsigned int grid_pos_x,
```

Comment:
string& sprite_path instead? Not specific to the fact this needs to reference a sprite.

Reply:
What's the difference between grid_pos_x and pos_x?

Reply:
Also does a GameObject need to have position values? For example when an item is in inventory it does not need position values. I'd also be ok if we just set -1, -1 for when off map as long as it is explicit.